packages/fxa-settings/src/components/UnitRowRecoveryKey/index.tsx

```diff
@@ -98,7 +95,16 @@ export const UnitRowRecoveryKey = () => {
         Why does resetting my password reset my data?
       </LinkExternal>
       {modalRevealed && (
-        <VerifiedSessionGuard onDismiss={hideModal} onError={onError}>
+        <VerifiedSessionGuard
+          onDismiss={hideModal}
+          onError={(error) => {
+            hideModal();
+            alertBar.error(
+              'Sorry, there was a problem verifying your session',
+              error as ApolloError
```

Comment:
Why'd we need `error as ApolloError` here and not elsewhere?

Reply:
I would guess this is casting an `AuthenticationError` to an `ApolloError`? maybe we should just have gql-api always send down an `ApolloError`, like Ben suggested [here](https://github.com/mozilla/fxa/pull/6504#discussion_r492215731)

Reply:
> Why'd we need error as ApolloError here and not elsewhere?

Huh, I did this because `ModalVerifySession`, by way of `VerifiedSessionGuard`, indicated it could return a regular `Error` or an `ApolloError`. But on closer inspection I don't see where `Error` is possible, so I've modified the function definition types.